### PR TITLE
Periods and Namespace

### PR DESCRIPTION
**Pull request by CoolMineman**

### CODE REVIEW

**Review by valoeghese — APPROVED** [review]
pls don't merge your own prs next time, unless they have a number of approvals

**Comment by valoeghese** [comment]
ok *now* you can merge your own pr :wink:

**Comment by Ueaj-Kerman** [comment]
translation: pls allow for sufficient :bike::house: befre merging pr

File: src/main/java/io/github/minecraftcursedlegacy/impl/networking/PluginChannelRegistryImpl.java
Comment (Ueaj-Kerman) [inline src/main/java/io/github/minecraftcursedlegacy/impl/networking/PluginChannelRegistryImpl.java:15]: no, just change the type in the hashmap

File: src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java
Comment (Ueaj-Kerman) [inline src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java:39]: change the ctor

**Review by Ueaj-Kerman — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/minecraftcursedlegacy/impl/networking/PluginChannelRegistryImpl.java
Comment (CoolMineman) [reply src/main/java/io/github/minecraftcursedlegacy/impl/networking/PluginChannelRegistryImpl.java:15]: saves ram

File: src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java
Comment (CoolMineman) [reply src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java:39]: ?

File: src/main/java/io/github/minecraftcursedlegacy/impl/networking/PluginChannelRegistryImpl.java
Comment (valoeghese) [reply src/main/java/io/github/minecraftcursedlegacy/impl/networking/PluginChannelRegistryImpl.java:15]: oh I didn't catch that

File: src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java
Comment (Ueaj-Kerman) [reply src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java:39]: change the constructor of plugin message packet instead of tostringing it

Comment (CoolMineman) [reply src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java:39]: no

Comment (Ueaj-Kerman) [reply src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java:39]: do it or I'll slap you

Comment (CoolMineman) [reply src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java:39]: false

Comment (Ueaj-Kerman) [reply src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java:39]: 👏 do it or I'll slap u again

Comment (CoolMineman) [reply src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java:39]: http://i.imgur.com/h2aHRVa.png

**Pull request merged**
